content-test/common/reducers/equality.test.js

```diff
@@ -0,0 +1,35 @@
+const createStore = require("common/create-store");
+const {actions} = require("common/action-manager");
+const {createRows} = require("test/faker");
+
+describe("Reducer equality", () => {
+  let store;
+  function dispatchRows(type, length = 10) {
+    store.dispatch(actions.Response(type, createRows({length})));
+  }
+  beforeEach(() => {
+    store = createStore({logger: false});
+  });
+  it("should return a reference", () => {
+    assert.equal(store.getState(), store.getState());
+  });
+  it("should dispatch rows", () => {
```

Comment:
The title here suggests that the test wants to test the dispatchRows function above, which it obviously doesn't.  

It seems like this is really testing that TOP_FRECENT_SITES_RESPONSE maps to newState.TopSites.rows, but perhaps I'm misinterpreting.

I think something wants to change here; it _might_ just be the description of the test.